ShelbyFinancials/ReadMe.md

```diff
@@ -0,0 +1,145 @@
+![Kingdom First Solutions](https://user-images.githubusercontent.com/81330042/113191442-2d82f580-9223-11eb-9e65-81ae5bc740f6.png)
+
+
+# Shelby Financials Export
+*Tested/Supported in Rock version:  8.0-12.0*   
+*Created:  10/29/2018*  
+*Updated:  11/20/2019*
+
+## Summary
+
+
+
+Quick Links:
+
+- [What's New](#whats-new)
+- [Configuration](#configuration)
+- [Exporting to Shelby Financials](#exporting-to-shelby-financials)
+
+
+
+## What's New
+
+- The following new goodness will be added to your Rock install with this plugin:
+  - **New Block**: Shelby Financials Batch to Journal (added to the Batch Detail page on install) 
+  - **New Block**:  Shelby Financials Batches to Journal (added to the Shelby GL Batch Export page on install) 
+  - **New Account Attributes**: There are a number of new Account attributes that control where transactions are posted in Shelby Financials
+  - **New Transaction Attribute**: Transaction Project
+  - **New Page**: Projects (Finance > Administration > Projects)
+  - **New Page**: Shelby GL Export (Finance > Functions > Shelby GL Export)  
+  - **New Defined Type**: Financial Projects stores the Defined Values that designate what Project a transaction or batch should be associated with  
+  - **New Batch Attribute**: Date Exported  
+
+
+
+## Configuration
+
+#### Batch to Journal Block
+
+After install, the Shelby Financials Batch to Journal block was added to your Batch Details page. 
+
+![](https://user-images.githubusercontent.com/81330042/113193210-53a99500-9225-11eb-9066-85ebd714e6ae.png)
+
+
+
+
+
+
+
+
+![](Images/BatchToJournalProperties.png)
+
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;1</span>**Name** Block name
```

Comment:
```suggestion
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;1&nbsp;&nbsp;</span>**Name** Block name
```

Reply:
For context, each of these suggestions is just adding space after the number because they look a bit off to me and need a bit of help:   
![image](https://user-images.githubusercontent.com/2990519/114585475-ecd2a580-9c40-11eb-8b2d-66fa4622e176.png)